src/main/java/com/jc/tm/database/dao/TaskDaoImpl.java

```diff
@@ -1,5 +1,121 @@
 package com.jc.tm.database.dao;
 
-public class TaskDaoImpl{
+import com.jc.tm.database.Status;
+import com.jc.tm.database.entity.Task;
 
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Logger;
+
+public class TaskDaoImpl implements TaskDao{
+
+    //field for log
+    public static final Logger LOGGER = Logger.getLogger(TaskDaoImpl.class.getName());
+
+    //sql commands
+    private static final String INSERT_TASK = "INSERT INTO task (id, name, description, due_Dates, status) " +
+            "values (?, ?, ?, ?, ?)";
+    private static final String UPDATE_TASK = "UPDATE task SET name = ?, description = ?, due_Dates = ? where id = ?";
+    private static final String SELECT_ALL_TASK = "SELECT id, name, description, due_Dates, status FROM TASK";
+    private static final String SELECT_BY_ID_TASK = "select id, name, description, due_Dates, status from task where id = ?";
+    private static final String DELETE_TASK = "DELETE FROM task WHERE id = ?";
+
+    private static final String _id = "id";
```

Comment:
better way you use upper case for constants

Reply:
Sure